app/src/main.cpp

```diff
@@ -73,7 +75,7 @@ int main(int argc, char *argv[]) {
 
   // first setup qapp
   //QApplication *app = new QApplication(argc, argv);
-  mars::app::MyApp *app=NULL;
+
   if(simulation->needQApp) {
     mars::app::MyApp *app = new mars::app::MyApp(argc, argv);
```

Comment:
Creates a temporal mars::app::MyApp *app, the global mars::app::MyApp *app=NULL from line 38 stays NULL, and quit in line 42 is never called